src/main/java/com/example/bookmyshow/utills/AppUtils.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bookmyshow.utills;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.util.AntPathMatcher;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class AppUtils {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    public static String getEncodedString(String rawStr){
+        return new String(Base64.getEncoder().encode(rawStr.getBytes(StandardCharsets.UTF_8)));
+    }
+
+    public static String getDecodedString(String base64Str){
+        return new String(Base64.getDecoder().decode(base64Str.getBytes(StandardCharsets.UTF_8)));
+    }
+
+    public static boolean matchesPattern(String pattern, String requestUri) {
```

Comment:
Why this method used

Reply:
We are using it because the given uri matches with pattern or not